SpellCheck/pom.xml

```diff
@@ -1,15 +1,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+	<artifactId>spellcheck</artifactId>
+	<name>spellcheck</name>
+
 	<parent>
 		<groupId>org.opentestsystem.delivery</groupId>
 		<artifactId>student-parent</artifactId>
 		<version>2.1.2-SNAPSHOT</version>
 	</parent>
-  <groupId>AIR.SpellCheck</groupId>
-  <artifactId>spellcheck</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
-  <name>spellcheck</name>
-  
+
 	<repositories>
 	    <!--other repositories if any-->
 	    <repository>
```

Comment:
This repository is referencing a maven repository embedded in source code with the dk.dren:hunspell:1.3.2-SNAPSHOT artifact hard-coded into it.  This will need to be fixed before we can release TDS_Student.